### PR TITLE
Spell checked the admin guide

### PR DESCRIPTION
**Pull request by jrouzierinverse**
Description
-----------

Spell checked the admin guide

Delete branch after merge
-------------------------
NO

### CODE REVIEW

**Comment by jrouzierinverse** [comment]
Ran all the documentation files through a spell checker.
Please review

File: docs/PacketFence_PKI_Quick_Install_Guide.asciidoc [hunk 1]
```diff
@@ -28,7 +28,7 @@ Installation
 Step 1: Install the PKI
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-NOTE: The PacketFence PKI application is available on the PacketFence repository as the package "packetfence-pki". This is available for CentOS6, Debian7 and Ubuntu 12.04.
+NOTE: The PacketFence PKI application is available on the PacketFence repository as the package "packetfence-pki". This is available for CentOS 6, Debian 7 and Ubuntu 12.04.
```
Comment (louismunro) [inline docs/PacketFence_PKI_Quick_Install_Guide.asciidoc]: I wonder if this is actually correct now?

Comment (jrouzierinverse) [reply docs/PacketFence_PKI_Quick_Install_Guide.asciidoc]: What standard would you prefer to use.
* CentOSX
* CentOS-X
* CentOS X
My prefered one is 'CentOS X' however it really does not matter as it is just minutia.

Comment (whitx) [inline docs/PacketFence_PKI_Quick_Install_Guide.asciidoc]: Well it was for 6.4, havn't tested in devel, should still be.

Comment (louismunro) [reply docs/PacketFence_PKI_Quick_Install_Guide.asciidoc]: It's not so much the standard I wonder about, as the supported versions (e.g. Ubuntu).

Comment (jrouzierinverse) [reply docs/PacketFence_PKI_Quick_Install_Guide.asciidoc]: @fdurand correct me if I am wrong we have not made any major changes to packetfence-pki recently so this should be still correct as for supported packages.

Comment (fdurand) [reply docs/PacketFence_PKI_Quick_Install_Guide.asciidoc]: i have the package for centos7, i just have to upload it in extra repo

Comment (louismunro) [reply docs/PacketFence_PKI_Quick_Install_Guide.asciidoc]: So I would remove any reference to Ubuntu and add one to CentOS 7/RHEL 7.

Comment (louismunro) [reply docs/PacketFence_PKI_Quick_Install_Guide.asciidoc]: Other than that I see nothing objectionable.
It can be merged.

Comment (jrouzierinverse) [reply docs/PacketFence_PKI_Quick_Install_Guide.asciidoc]: Done

**Pull request merged**